docs/dagster-university/pages/dagster-dbt/lesson-3/3-representing-the-dbt-project-in-dagster.md

```diff
@@ -0,0 +1,35 @@
+---
+title: 'Lesson 3: Representing the dbt project in Dagster'
+module: 'dagster_dbt'
+lesson: '3'
+---
+
+# Representing the dbt project in Dagster
+
+As you’ll frequently point your Dagster code to the `target/manifest.json` file and your dbt project in this course, it’ll be helpful to keep a reusable representation of the dbt project. This can be easily done using the `DbtProject` class.
+
+Create a new file in the `assets` directory called `dbt.py`. Open that file and add the following import at the top:
+
+```python
+from pathlib import Path
+
+from dagster_dbt import DbtProject
+```
+
+The `Path` class from the `pathlib` standard library will help us create an accurate pointer to where our dbt project is. The `DbtProject` class is imported from the `dagster_dbt` package that we installed earlier. After the import, add the following code: 
+
+```python
+dbt_project = DbtProject(
+  project_dir=Path(__file__).joinpath("..", "..", "..", "analytics").resolve(),
+)
+```
+
+This code creates a representation of the dbt project called `dbt_project`. The code defining the location of the project directory might look a little complicated, so let’s break it down:
+
+- It uses the location of the `dbt.py` file (via `__file__`) as a point of reference for finding the dbt project
```

Comment:
Nit - what does `it` in `It uses the location of...` refer to here? The code? Something specifically in the code?

Reply:
Rephrased in [54fd52f](https://github.com/dagster-io/dagster/pull/23098/commits/54fd52f9d9ea4631d085df33a7dccb2eb41d47d6) to `The location of ... is used as a point ...`